src/main/resources/l10n/Menu_de.properties

```diff
@@ -211,3 +211,5 @@ Disable_highlight_groups_matching_entries=Keine_Gruppen_hervorheben
 Online_help_forum=Online-Hilfeforum
 
 Manage_protected_terms=
+
+BibTeX=
```

Comment:
All format names like BibTeX or BibLaTeX do not need to be translated, same applies to things like the fetcher names (e.g. Springer, Gvk ), as this are usually proper nouns.